mne/dipole.py

```diff
@@ -346,6 +346,14 @@ class DipoleFixed(object):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
+    pos : array, shape (n_dipoles, 3)
+        The dipoles positions (m) in head coordinates.
+    amplitude : array, shape (n_dipoles,)
+        The amplitude of the dipoles (Am).
+    ori : array, shape (n_dipoles, 3)
+        The dipole orientations (normalized to unit length).
+    gof : array, shape (n_dipoles,)
+        The goodness of fit.
```

Comment:
We don't want to add these as attributes. They won't be saved (not part of the FIF format spec for this file type), so they won't survive an I/O round trip.